# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/jimmy/mplayer/no-altivec.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./cpudetect.c.orig	2004-01-21 14:52:42.000000000 +0100
+++ ./cpudetect.c	2004-01-21 14:57:28.000000000 +0100
@@ -551,6 +551,7 @@
 /* no Darwin, do it the brute-force way */
 /* this is borrowed from the libmpeg2 library */
         {
+	/*
           signal (SIGILL, sigill_handler);
           if (sigsetjmp (jmpbuf, 1)) {
             signal (SIGILL, SIG_DFL);
@@ -565,6 +566,7 @@
             signal (SIGILL, SIG_DFL);
             caps->hasAltiVec = 1;
           }
+	*/
         }
 #endif /* SYS_DARWIN */
         mp_msg(MSGT_CPUDETECT,MSGL_INFO,"AltiVec %sfound\n", (caps->hasAltiVec ? "" : "not "));
